### DISCUSSION TITLE
Weird EOL characters in side-by-side diff output ( ^[[0K )

### DISCUSSION THREAD

**Discussion by 2art** [dandavison/delta#1695] [General]
The side-by-side output right side is filled with this ^[[0K character at ends of lines. I tried playing around with settings a lot to no avail. See picture for what I mean..

.gitconfig includes:

```gitconfig
[core]
	eol = lf
	autocrlf = input
	editor = "vscodium --wait"
	pager = delta
[interactive]
	diffFilter = delta --color-only --features=interactive
[pager]
	diff = delta
[delta]
	side-by-side = true
[diff]
	tool = delta
```

![ss-ddmeos240422-240506-211409-1640x952](https://github.com/dandavison/delta/assets/5929182/be94b550-dcd8-40d1-9685-ab7f9ec29ee6)

**Comment by dandavison** [comment]
Hi @2art, problems like this are due to your terminal emulator, or `less` version. What OS and terminal emulator are you using, and what version of `less`?

**Comment by th1000s** [comment]
This seems very related: [Getting "^\[\[0K" at end of some lines](https://github.com/dandavison/delta/discussions/1630).
